### PR TITLE
[DX-808] Update the url of the mdcb release notes to the new path

### PR DESCRIPTION
**Pull request by letzya**
Move the file of mdcb release notes to the correct place in the new navigation. Added alias to make sure we dont get 404

[808](https://tyktech.atlassian.net/browse/DX-808) For internal users - Please add a Jira DX PR ticket to the subject!

<br>

---

### Preview Link
https://deploy-preview-3561--tyk-docs.netlify.app/docs/nightly/release-notes/mdcb-2.4/

<br>

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

### CODE REVIEW

File: tyk-docs/data/menu.yaml [hunk 1]
```diff
@@ -3130,30 +3130,16 @@ menu:
...
-        - title: "Release highlights and upgrades"
-          category: Directory
-          show: False
-          menu:
-          - title: "[version]"
-            category: Page
-            show: False
         - title: "MDCB v2.4"
-          path: /release-notes/mdcb-2.4
+          path: /product-stack/tyk-enterprise-mdcb/release-notes/version-2.4.md
```
Comment (letzya) [inline tyk-docs/data/menu.yaml]:
```suggestion
          path: /product-stack/tyk-enterprise-mdcb/release-notes/version-2.4
```

**Review by dcs3spp — COMMENTED** [review]
Hi @letzya ,is it only the MDCB 2.4 release note for this ticket or should MDCB 2.3, 2.2 etc. be moved too?

**Comment by letzya** [comment]
@dcs3spp You could but no need to block releasing this one. 
I did it so the next release will be added to the correct place.

**Pull request merged**